backend/globaleaks/utils/file_analysis/ScanData.py

```diff
@@ -0,0 +1,11 @@
+from globaleaks.utils.file_analysis.VirusScanResult import VirusScanResult
```

Comment:
Tests seems to be missing for the antivirus module both on the backend and on the client